### PR TITLE
Img view fix

### LINKED ISSUE #696: Incorrect georef is added to uploaded images
All images recently uploaded to the demo (including simple screenshots like http://c2corgv6.demo-camptocamp.com/images/737030/fr/asdads) surprisingly have a geometry, always set to some point in Wallis!

When uploading the image above I have just typed a random title and pick "collaborative" as type. That's all.

Where does this geometry come from?

### PR DESCRIPTION
**Pull request by ginold**
fixes #696

### CODE REVIEW

File: c2corg_ui/templates/image/view.html [hunk 1]
```diff
@@ -33,21 +35,34 @@
...
+      "features": [
+      {
+        "type": "Feature",
+        "geometry": {"type": "Point", "coordinates": [${geometry.x}, ${geometry.y}]},
+        "properties": {
+          "title": ${dumps(locale['title']) | n},
+          "lang": "${locale['lang']}",
+          "documentId": ${image_id},
+          % if image['elevation'] is not None:
+            "elevation": ${image['elevation']},
```
Comment (asaunier) [inline c2corg_ui/templates/image/view.html]: There's no elevation for images. At least it's not a relevant info, no need to pass it to the FeatureCollection.

File: c2corg_ui/templates/image/view.html [hunk 2]
```diff
@@ -33,21 +35,34 @@
...
+          "module": "images",
+          "highlight": true
+        }
       }
-    }]
-  });
-
+      % if not version:
+        ${associated_waypoints_featurelist(waypoint)}
+      % endif
```
Comment (asaunier) [inline c2corg_ui/templates/image/view.html]: You can skip this part too. Anyway I don't think ``waypoint`` is the correct argument name.

File: c2corg_ui/templates/image/view.html [hunk 3]
```diff
@@ -17,9 +17,11 @@
...
 
 <%
-image_id = image['document_id']
-image['doctype'] = 'images'
-other_langs, missing_langs = get_lang_lists(image, lang)
+  image_id = image['document_id']
+  image['doctype'] = 'images'
+  other_langs, missing_langs = get_lang_lists(image, lang)
+  ##geometry4326 = transform(geometry, 'epsg:3857', 'epsg:4326')
+  geometry4326 = geometry
```
Comment (asaunier) [inline c2corg_ui/templates/image/view.html]: Is ``geometry4326`` really used in the template?

**Comment by ginold** [comment]
done! will merge when travis passes.

**Pull request merged**